gradle-consistent-versions-idea-plugin/src/main/java/com/palantir/gradle/versions/intellij/RepositoryExplorer.java

```diff
@@ -158,14 +193,21 @@ private boolean isStableVersion(String version) {
                 .matches();
     }
 
-    @Value.Immutable
-    interface CacheKey {
-        String url();
+    private void triggerRefresh() {
```

Comment:
We want to trigger the refresh at this level then we can refresh each time something is added to the cache